ecs-logs-collector.sh

```diff
@@ -567,13 +591,24 @@ enable_ecs_agent_debug() {
       then
         info "Debug mode is already enabled."
       else
-        if [ -e /etc/ecs/ecs.config ]; then
-          echo "ECS_LOGLEVEL=debug" >> /etc/ecs/ecs.config
+        echo "ECS_LOGLEVEL=debug" >> /etc/ecs/ecs.config
 
-          try "restart the Amazon ECS Container Agent to enable debug mode"
-          stop ecs; start ecs
-        fi
+        try "restart the Amazon ECS Container Agent to enable debug mode"
+        systemctl restart ecs
```

Comment:
AL AMI will need to keep the upstart instructions to restart the Agent:
```suggestion
        stop ecs; start ecs
```

Reply:
Yep, that's right.